lib/pod/command/trunk/push.rb

```diff
@@ -119,7 +121,10 @@ def validate_podspec
           validator = Validator.new(spec, %w(https://github.com/CocoaPods/Specs.git))
           validator.allow_warnings = @allow_warnings
           validator.use_frameworks = @use_frameworks
-          validator.swift_version  = @swift_version if Validator.method_defined?(:swift_version=)
+          if Validator.method_defined?(:use_modular_headers=)
```

Comment:
`if validator.respond_to?(:use_modular_headers=)`

Reply:
could also change the above too?

Reply:
👍

Reply:
also the reason I used a separate `if` check instead of inline was for rubocop line length